README.md

```diff
@@ -1,38 +1,74 @@
 # IWD dev team ladder
 
+This job ladder exposes all roles inside IWD development department, and define the goals, expectations and responsibilities of each role.
+Constant progression, and meeting your individual objectives will help you to climb it.
```

Comment:
To climb this ladder you know what to do 😄

Reply:
Fixed